docs/RELEASE.md

```diff
@@ -0,0 +1,40 @@
+# Release process
+
+When we want to do a new release, push a git tag with format `v**` and workflow `releases.yml` executes.
+
+This workflow internally uses [go-releaser](https://goreleaser.com/ci/actions/) to push a new release.
+
+Please follow the below detailed steps.
+- Create a tag in format `v202205240715`
+```code
+git tag -a `date "+v%Y%m%d%H%M"` -m "Release a new version"
+```
+- Get the tag name
+```code
+git tag
+```
+- Push the newly created tag
+```code
+git push origin <tag>
+```
+- Check workflow `releases.yml` should trigger
+- Once completed, go to repo releases page and edit the newly created release as `pre-release`, so we can do sanity testig before we officicaly release
```

Comment:
Could you link to the repo releases page for convenience.

Reply:
fixed and also added link to release workflow